### PR TITLE
fix(alternator): added chaos monkey nemesis and 2 more loaders

### PR DESCRIPTION
**Pull request by fgelcer**
was decided that we needed to increase from 1 to 3 loaders, to generate
enough load.
also, there was no nemesis, so was decided to add some nemesis, and for
last run, there were no issues with nemesis (the ones that ran), so if
any nemesis has problems, we will either skip it (if alternator then
skip) or to remove it from alternator nemesis list.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by bentsi** [comment]
@fgelcer scope of the commit message is incorrect. Please read the "best practices" link in the PR self-review

**Comment by fruch** [comment]
@fglacer we have some results to share if the run with the nemsis ?

**Comment by fgelcer** [comment]
> @fgelcer scope of the commit message is incorrect. Please read the "best practices" link in the PR self-review

changed

**Comment by fgelcer** [comment]
> @fglacer we have some results to share if the run with the nemsis ?

as i said in the description of the PR... last run had no issues, but i still not 100%, as not all nemesis were called.

**Comment by fgelcer** [comment]
@bentsi / @fruch

File: test-cases/longevity/longevity-alternator-3h.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ stress_cmd: ["bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p reco
...
 dynamodb_primarykey_type: HASH_AND_RANGE
 
-n_loaders: 1
+n_loaders: 3
 instance_type_db: 'i3.2xlarge'
 n_monitor_nodes: 1
 n_db_nodes: 3
 
-nemesis_class_name: NoOpMonkey # ChaosMonkey
+nemesis_class_name: ChaosMonkey
```
Comment (bentsi) [inline test-cases/longevity/longevity-alternator-3h.yaml]: better to put strings in quotes ''

**Comment by bentsi** [comment]
let's see how it works on master and then backport

**Comment by fgelcer** [comment]
> let's see how it works on master and then backport

@bentsi , it works (although there was a failure), so ready to backport.

**Pull request merged**